.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,23 @@
+name: E2E tests
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      E2E_TESTS: true
+      E2E_LOGS: true
+      CI_VERBOSE: true
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Run tests
+        run: make test-e2e
+      - name: Archive test logs
+        if: always()
+        uses: actions/upload-artifact@v3
```

Comment:
Should we set retention time? Maybe delete log artifacts older than 30 days ?

Reply:
Didn't know that was possible :), 30 looks like more than enough, added!